dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/imports/bundle/TrackerObjectsMapperTest.java

```diff
@@ -582,11 +584,15 @@ private void assertMappedEvent(
     assertNotes(event.getNotes(), actual.getNotes(), updatedBy);
   }
 
+  private void assertUids(UID uid, UidObject uidObject) {
```

Comment:
I think this is useful to have in `dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/test/utils/Assertions.java`. Maybe as `assertEqualUids`